qa/rpc-tests/test_framework/test_framework.py

```diff
@@ -5,7 +5,7 @@
 
 # Base class for RPC testing
 
-# Add python-bitcoinrpc to module search path:
+# Add python-zcashrpc to module search path:
```

Comment:
Is this correct?

Reply:
Reverted with same caveats as above.  Should I rebase?

Reply:
No.  It's obsolete, and removed.  See above.